use-ci-nexus-cache/action.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Use CI Nexus cache
+
+description: Sets up use of co-located pull-through cache for Maven artifacts via ~/.m2/settings.xml.
+
+inputs:
+  server_username:
+    description: 'Nexus Username'
+  server_password:
+    description: 'Nexus Password'
+  server_id:
+    description: 'server ID'
+
+runs:
+  using: composite
+  steps:
+  - name: Create ~/.m2/settings.xml with cache configuration
+    shell: bash
+    run: |
+      cat <<EOF > ~/.m2/settings.xml
+      <?xml version="1.0" encoding="UTF-8"?>
+      <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+        <interactiveMode>false</interactiveMode>
+        <servers>
+          <server>
+            <id>${{ inputs.server_id }}</id>
+            <username>${{ inputs.server_username }}</username>
+            <password>${{ inputs.server_password }}</password>
+          </server>
+        </servers>
+        <mirrors>
+          <mirror>
+            <id>${{ inputs.server_id }}</id>
+            <mirrorOf>*</mirrorOf>
+            <name>Camunda Nexus</name>
+            <url>https://repository.nexus.camunda.cloud/content/groups/internal/</url>
```

Comment:
does it make sense to make this dynamic as well, with a default value set to `https://repository.nexus.camunda.cloud/content/groups/internal/`? (e.g. like we do with `server_id`). Not sure if it makes sense, just was thinking about it when seeing `server_id` dynamic.

Reply:
nvm, found answer in https://github.com/camunda/team-infrastructure/issues/282#issuecomment-1406390331